perf/prometheus-install/values.yaml

```diff
@@ -1,325 +0,0 @@
-fullnameOverride: "istio"
```

Comment:
Where is the scrape config now?
Is any of this going to work with the old installer ?

Reply:
the scrape config is included by the new installer part - it sets up the ServiceMonitors, etc

Reply:
before, we used the old installer + fully custom Prometheus, now we use the old installer + a la carte Prometheus operator from the new installer. it still works (once a small pr to installer is merged)